packages/kbn-mapbox-gl/src/index.ts

```diff
@@ -35,7 +35,7 @@ import 'mapbox-gl/dist/mapbox-gl.css';
 mapboxgl.workerUrl = mbWorkerUrl;
 mapboxgl.setRTLTextPlugin(mbRtlPlugin);
 
-export {
+export type {
   mapboxgl,
```

Comment:
I'm wondering if we can make this a type export only. Some of these are actually constructor functions that are used in the Maps and Vega plugins.

Reply:
Maybe then we need to restructure the import exports here? Previously only types were being imported for those values, but everything was being exported which Babel complains about. 

I've restructured it and exported as types what is being imported as types only, and code exporting what is being imported as code. Hope that works

Reply:
@thomasneirynck could you approve the PR if we are okay now? 😄